examples/bigquery-snowflake-tables-migration-utility/config/sf-ddl-extraction-config.json

```
@@ -0,0 +1,31 @@
+{
+    "table_config": [
+        {
+            "table_name": "PUBLIC.CALL_CENTER",
+            "clustering_fields": "CC_CALL_CENTER_ID",
+            "partitioning_field": "CC_REC_START_DATE"
+
+        },
+        {
+            "table_name": "PUBLIC.WEB_SITE",
+            "clustering_fields": "WEB_SITE_ID",
+            "partitioning_field": "WEB_REC_START_DATE"
+
+        }
+
+
+    ],
+    "project_id" : "poc-env-aks" ,
```

please restrain from adding names of actual google projects/objects in public repos. Please replace with placeholder 
```suggestion
    "project_id" : "<my-project>" ,
```